data/mods/Magiclysm/Spells/classless.json

```
@@ -1,4 +1,22 @@
 [
+  {
+    "id": "nightvision",
+    "type": "SPELL",
+    "name": "Dark Sight",
+    "description": "Gives you the power to see in the dark",
+    "message": "Your eyes glow green for a second and suddenly your sight can pierce the darkest shadows.",
+    "valid_targets": [ "self" ],
+    "effect": "target_attack",
+    "effect_str": "night_vision",
+    "min_range": 1,
+    "max_range": 1,
+    "base_casting_time": 100,
+    "base_energy_cost": 100,
```

```suggestion
    "base_energy_cost": 400,
```
i prefer to err on the side of "a bit too weak" but i think 400 should be a good number for the length